lib/id/cli.js

```diff
@@ -0,0 +1,40 @@
+/*!
+ * Copyright (c) 2021 Digital Bazaar, Inc. All rights reserved.
+ */
+import * as id from '.';
+import {_displayOutput} from '../displayOutput.js';
+import {_setupFormatOption, _initSecretKeySeed} from '../options.js';
+
+function idCommand(yargs) {
+  return yargs.command('id <action>', 'Generate and manage DIDs.', {
+    handler: _handler,
+    builder: _yargs => {
+      _setupFormatOption(yargs);
+      return _yargs
+        .positional('action', {
+          describe: 'Action to perform.',
+          choices: ['create'],
+          default: 'create'
+        })
+        .option('method', {
```

Comment:
Do we need this `method` option if the only supported method is `did:key`?

Reply:
At the moment, we don't need it, strictly. But I do plan to add support for did:v1 and did:web in the next PRs, so might as well leave it, I think.